packages/css-size/src/__tests__/index.js

```diff
@@ -3,7 +3,6 @@ import { spawn } from 'child_process';
 import path from 'path';
 import * as assert from 'uvu/assert';
 import { test } from 'uvu';
-import colors from 'colors/safe';
```

Comment:
Look like a `chalk` ancestor that  somehow still ends up on the module resolution path, better to drop it completely rather than add a new dependency.